README.md

```diff
@@ -35,14 +50,14 @@ The IntelliJ Ballerina plugin provides the Ballerina development capabilities in
 
 Below you can see the plugin versions that correspond to the versions of the IntelliJ Platform.
 
-**Plugin Version**|**Platform Version Compatibility**
-:-----:|:-----:
-0.8.0 - 0.8.2 | IntelliJ IDEA 2016.3 - 2016.4
-0.8.3 - 0.981.0 | IntelliJ IDEA 2016.3 - 2017.2
-0.982.0 - 0.991.0 | IntelliJ IDEA 2017.3 - 2018.2
-0.991.1 - 1.2.1 | IntelliJ IDEA 2018.3 - 2019.3
-1.2.2 - 1.2.x | IntelliJ IDEA 2018.3 - 2020.2
-2.0.0 | IntelliJ IDEA 2022.0 - 2024.1
+| **Plugin Version** | **Platform Version Compatibility** |
+|:------------------:|:----------------------------------:|
+|   0.8.0 - 0.8.2    |   IntelliJ IDEA 2016.3 - 2016.4    |
+|  0.8.3 - 0.981.0   |   IntelliJ IDEA 2016.3 - 2017.2    |
+| 0.982.0 - 0.991.0  |   IntelliJ IDEA 2017.3 - 2018.2    |
+|  0.991.1 - 1.2.1   |   IntelliJ IDEA 2018.3 - 2019.3    |
+|   1.2.2 - 1.2.x    |   IntelliJ IDEA 2018.3 - 2020.2    |
+|       2.0.0        |   IntelliJ IDEA 2022.0 - 2024.1    |
 
 ## Useful links
 * Use the ballerina-dev@googlegroups.com mailing list to discuss code changes to the Ballerina project.
```

Comment:
This is invalid. Use discord instead.

Reply:
Updated in https://github.com/ballerina-platform/plugin-intellij/pull/77/commits/60e2691677e161f6e34b89dede063c7035fa74f0